### PR TITLE
WIP: Using SplFileObject::getSize instead of UploadedFile::getClientSize

### PR DESCRIPTION
**Pull request by Adapik**
solves #41

### CODE REVIEW

**Comment by Adapik** [comment]
@weaverryan It's inherited from \SplFileInfo, exists in PHP since PHP 5.1.2

**Comment by fabpot** [comment]
Tests are apparently broken.

File: .travis.yml [hunk 1]
```diff
@@ -20,6 +20,8 @@ matrix:
         - php: 5.6
           env: COMPOSER_OPTIONS="" SYMFONY_DEPRECATIONS_HELPER=weak
         - php: 7.0
+        - php: 7.1
+        - php: 7.2
         - php: hhvm
     allow_failures:
         - php: hhvm
```
[the remark this answers is not in the document]
Comment (Adapik) [reply .travis.yml]: Yes, i'll remove it

**Comment by Adapik** [comment]
I have fixed failing tests. Some deprecation notices remaining: 
`Passing a size as 4th argument to the constructor of "Symfony\Component\HttpFoundation\File\UploadedFile" is deprecated since Symfony 4.1: 8x` in `HttpFoundationFactory::createUploadedFile` and tests. Should we fix it too in this PR? Any ideas how to support 2 versions of `UploadedFile:__construct()`?

**Comment by Nyholm** [comment]
This also is fixed in #45

**Comment by Nyholm** [comment]
This is can be closed

**Pull request closed without merging**